website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -16,6 +16,7 @@ dbt commands are run against dbt Cloud's infrastructure and benefit from:
 * Support for dbt Mesh ([cross-project `ref`](/docs/collaborate/govern/project-dependencies)),
 * Significant platform improvements, to be released over the coming months.
 
+<Lightbox src="/img/docs/dbt-cloud/cloud-cli-overview.jpg" title="Overview of how the dbt Cloud CLI works with dbt Cloud's infrastructure to run dbt commands from your local command line." />
```

Comment:
For accessibility purposes (so the reader is clear on what is in the picture) this might be helpful
```suggestion
<Lightbox src="/img/docs/dbt-cloud/cloud-cli-overview.jpg" title="Diagram of how the dbt Cloud CLI works with dbt Cloud's infrastructure to run dbt commands from your local command line." />
```